agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml

```diff
@@ -134,6 +137,9 @@ spec:
 #        - mountPath: /host/etc/VERSION
 #          name: etc-version
 #          readOnly: true
+        - mountPath: /host/etc
```

Comment:
With this mount, L28-32 and L136-139 isn't needed anymore, in this and the agent-slim yaml

Reply:
Would they do any harm? I wanted to keep them there for convenience, in case the user wants to drop the /etc one.

Reply:
Fair enough, I've left them in